docs/changes/visualization/17175.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Fixed a bug that caused ``CoordinateHelper.get_axislabel()`` to not return
+anything instead of the default label if no label has been explicitly
```

Comment:
```suggestion
Fixed a bug that caused ``CoordinateHelper.get_axislabel()`` to return an
empty string instead of the default label if no label has been explicitly
```